### DISCUSSION TITLE
How can i create a global function and call it into a Livewire component?

### DISCUSSION THREAD

**Discussion by FrancissEscamez** [livewire/livewire#4817] [1. Help] [answered]
Hi, i had a simple livewire component to make a list with a search bar, and sorting <th>, i had this function that change my values and reset to page 1
```
    public function order($sort){
        if($this->sort == $sort){
            if($this->direction == 'desc'){
                $this->direction = "asc";
            }
            else{
                $this->direction = "desc";
            }
        }else{
            $this->direction = 'asc';
        }
        $this->sort = $sort;
        $this->resetPage();
    }
```
but i want to use it in all my lists, without declaring the same function in all my components, i try to put it in a custom helper but i cant call the helper inside the livewire component, how can i do this?

**Comment by Martleby** [comment, marked as answer]
I'd put it in a trait - then import and use the trait in your components.

**Reply by FrancissEscamez** [reply]
I look for trait doc and it work perfectly, Thanks!
